01_Data/src/layers/sequelize/repository/TransactionEvent.ts

```diff
@@ -193,4 +193,8 @@ export class SequelizeTransactionEventRepository extends SequelizeRepository<Tra
       })
       .then((row) => row as MeterValue[]);
   }
+
+  findByTransactionId(transactionId: string): Promise<Transaction | undefined> {
+    return this.transaction.readOnlyOneByQuery({ where: { transactionId } });
```

Comment:
We need both transactionId and stationId to find only one transaction since in OCPP 2.0.1, transactionId is generated by each charging station. To find a station id, I think we can use session id and search in session and location when these 2 modules are ready. 🤔

Reply:
Gotcha. I see in OCPP spec that it says this ID only needs to be unique for the charging station.